src/frontend/src/instance/opentsdb.rs

```diff
@@ -27,23 +27,27 @@ use crate::instance::Instance;
 
 #[async_trait]
 impl OpentsdbProtocolHandler for Instance {
-    async fn exec(&self, data_point: &DataPoint, ctx: QueryContextRef) -> server_error::Result<()> {
+    async fn exec(
+        &self,
+        data_points: &[DataPoint],
```

Comment:
We can also make data_points become Vec<DataPoint>, then we can directly move it into `data_point_to_grpc_row_insert_requests`